source/examples/rapids-sagemaker-higgs/Dockerfile

```diff
@@ -2,14 +2,21 @@ ARG RAPIDS_IMAGE
 
 FROM $RAPIDS_IMAGE as rapids
 
-# add sagemaker-training-toolkit [ requires build tools ], flask [ serving ], and dask-ml
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential \
-    && source activate rapids \
-    && pip3 install sagemaker-training cupy-cuda11x flask \
-    && pip3 install --upgrade protobuf
+# Installs a few more dependencies
+RUN conda install --yes -n base \
+        cupy \
+        flask \
+        protobuf \
+        sagemaker
```

Comment:
`apt-get` installing stuff is not allowed in the `rapidsai/core` docker images: https://github.com/rapidsai/docker/issues/583

Other things this fixes:

* the comment was wrong (`dask-ml` is not installed here)
* `sagemaker-training` is unnecessary